src/locales/en.json

```diff
@@ -2,17 +2,15 @@
   "404": {
     "title": "The content you’re looking for seems to have disappeared.",
     "main": "Go to {link} or search for something similar from the field below.",
-    "link-title": "Openverse",
     "search-placeholder": "Search for content to reuse"
   },
   "hero": {
-    "brand": "Openverse",
     "subtitle": "Browse through over 600 million items to reuse",
     "search": {
       "placeholder": "Search for content"
     },
     "disclaimer": {
-      "content": "All Openverse content is under a {license} or is in the public domain.",
+      "content": "All {openverse} content is under a {license} or is in the public domain.",
```

Comment:
There's a second instance of this string that needs updating as well: 

https://github.com/WordPress/openverse-frontend/blob/bec564f99b986015a018269363e32860d28f20bd/src/pages/index.vue#L118